assets/blocks/index.js

```diff
@@ -13,24 +13,46 @@ const attributes = {
 registerBlockVariation( 'core/button', {
 	name: 'sensei-certificates/view-certificate-button',
 	title: __( 'View Certificate', 'sensei-certificates' ),
-	description: __( 'Enable a learner to view their course certificate.', 'sensei-certificates' ),
-	keywords: [
-		__( 'Certificates', 'sensei-lms' ),
-	],
+	description: __(
+		'Enable students to view their course certificate.',
+		'sensei-certificates'
+	),
+	keywords: [ __( 'Certificates', 'sensei-lms' ) ],
 	category: 'sensei-lms',
 	attributes,
 	isActive: ( blockAttributes, variationAttributes ) =>
 		blockAttributes.className?.match( variationAttributes.className ),
 } );
 
-const addBlockToTemplate = ( blocks ) => ( [
+const addBlockToTemplate = ( blocks ) => [
 	...blocks,
-	[ 'core/button', attributes ]
-] );
+	[ 'core/button', attributes ],
+];
 
 // Add this block to the Course Completed Actions block.
 addFilter(
 	'sensei-lms.Course.completedActions',
 	'sensei-certificates',
 	addBlockToTemplate
 );
+
+registerBlockVariation( 'core/buttons', {
+	name: 'sensei-certificates/view-certificate-buttons',
+	title: __( 'View Certificate', 'sensei-certificates' ),
```

Comment:
I'm thinking about this name. As you said, this part could cause some confusion for the users. But maybe it could help if we use another name like "Buttons with View Certificate". WDYT?

If you agree, maybe we could tweak the description too.

Reply:
I have been also struggling with the name. I will try what you suggested below and leave this as a normal `core/buttons`.